www_src/components/color-group/color-group.jsx

```diff
@@ -38,11 +38,6 @@ var ColorGroup = React.createClass({
           <input className="sr-only" name="color" type="radio" value={color} checked={this.valueLink.value === color ? true : null} onChange={this.onChange} />
         </label>
       })}
-      <div className="tinker-container">
```

Comment:
actually, what do you think about just hiding this instead? like with style display none or something. then we dont have to dig through the commit logs to reimplement